src/site/facilities/service_location.drupal.liquid

```diff
@@ -70,11 +83,15 @@
   {% endif %}
 
   {% if locationEntity.fieldPhoneNumbersParagraph != empty %}
-    <div class="phone-numbers" data-template="paragraphs/service_location">
-      <h5>Phone</h5>
+    <div class="vads-u-display--flex vads-u-flex-direction--column" data-template="paragraphs/service_location">
+      {% if haveLocationName %}
+        <h4 class="force-small-header">Phone</h4>
+      {% else %}
+        <h5>Phone</h5>
+      {% endif %}
       {% if locationEntity.fieldPhoneNumbersParagraph %}
         {% for number in locationEntity.fieldPhoneNumbersParagraph %}
-          <a class="vads-u-margin-bottom--1" aria-label="{{ number.entity.fieldPhoneNumber | accessibleNumber }}"
+          <a aria-label="{{ number.entity.fieldPhoneNumber | accessibleNumber }}"
```

Comment:
where does `accessibleNumber` come from ? I can't find it assigned above.

Reply:
It's a filter function defined [here](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/filters/liquid.js#L469).